src/liquid-options.ts

```diff
@@ -89,11 +89,13 @@ export function normalize (options?: LiquidOptions): NormalizedOptions {
   if (options.hasOwnProperty('root')) {
     options.root = normalizeStringArray(options.root)
   }
-  let cache: Cache<Template[]> | undefined
-  if (typeof options.cache === 'number') cache = options.cache > 0 ? new LRU(options.cache) : undefined
-  else if (typeof options.cache === 'object') cache = options.cache
-  else cache = options.cache ? new LRU<Template[]>(1024) : undefined
-  options.cache = cache
+  if (options.hasOwnProperty('cache')) {
+    let cache: Cache<Template[]> | undefined
+    if (typeof options.cache === 'number') cache = options.cache > 0 ? new LRU(options.cache) : undefined
+    else if (typeof options.cache === 'object') cache = options.cache
+    else cache = options.cache ? new LRU<Template[]>(1024) : undefined
+    options.cache = cache
+  }
```

Comment:
Is this change necessary?

Reply:
Yes, the change in [liquid.ts#78](https://github.com/harttle/liquidjs/blob/90b863104e721a6041cf9d0f75de99d0d2767dda/src/liquid.ts#L78) would make couple of other tests crash. The options-object would always have the cache-key set to undefined which would overwrite the base cache.
I added two tests (sync/async) to demonstrate and test this problem